teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -284,6 +284,16 @@ public class P2POptions {
       hidden = true)
   private int batchVerifyMaxBatchSize = P2PConfig.DEFAULT_BATCH_VERIFY_MAX_BATCH_SIZE;
 
+  @Option(
+      names = {"--Xp2p-dumps-to-file-enabled"},
+      paramLabel = "<BOOLEAN>",
+      showDefaultValue = Visibility.ALWAYS,
+      description = "Enables saving P2P dumps to file.",
```

Comment:
Save objects to file that cause problems when processing, for example rejected blocks or invalid gossip.